content/02.main-text.md

```diff
@@ -442,7 +442,7 @@ These guidelines did not discuss broader community norms that may have improved
 It is also important to consider how the move to an open contribution model affects under-represented minority members of the scientific community [@url:https://lgatto.github.io/open-and-open/].
 Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1109/hicss.2007.277].
 <!-- cited a previous version of doi:10.1080/08874417.2009.11645338 (see https://git.io/vdSSR) -->
-Conferences and open source projects have used codes of conduct to establish these norms [@url:http://geekfeminism.wikia.com/wiki/Code_of_conduct_evaluations?oldid=35801; @url:https://www.contributor-covenant.org/].
```

Comment:
I'd like to keep the contributor covenant reference as a way of using a relevant primary source and giving credit to its creator.  I'm open to removing the geekfeminism link and replacing it with the IEEE paper, which appears to have a similar goal but with more data.

Reply:
In https://github.com/greenelab/meta-review/pull/211/commits/c40e2fa444adeaded76d53fdcfad362dfb475175, I readded Contributor Covenant as a link rather than a citation. It doesn't make much sense IMO to cite a homepage that is likely to change over time. I considered citing version 4.1 of the covenant, but that seemed too narrow (also didn't include a date or author list), so I think it just makes sense to link to the homepage.